disasm/isa_parser.cc

```diff
@@ -314,6 +320,31 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
       extension_table[EXT_ZICFILP] = true;
     } else if (ext_str == "zicfiss") {
       extension_table[EXT_ZICFISS] = true;
+    } else if (ext_str.substr(0, 3) == "zvl") {
+      try {
+        vlen = std::stol(ext_str.substr(3, ext_str.size() - 4));
+      } catch (std::logic_error& e) {
+        vlen = 0;
+      }
+      if ((vlen & (vlen - 1)) != 0 || vlen < 32)
+        bad_isa_string(str, ("Invalid Zvl string: " + ext_str).c_str());
```

Comment:
Strictly speaking, something like `V_Zvl64b` is legal.  Zvl64b means VLEN >= 64, not VLEN = 64; similarly, Zve64f doesn't mean that Zve64d is absent.  I think we can use Zvl to increase VLEN, rather than overriding it, as in something like

```
      int new_vlen = 0;
      try {
        new_vlen = std::stol(ext_str.substr(3, ext_str.size() - 4));
      } catch (std::logic_error& e) {}
      if ((new_vlen & (new_vlen - 1)) != 0 || new_vlen < 32)
        bad_isa_string(str, ("Invalid Zvl string: " + ext_str).c_str());
      vlen = std::max(vlen, new_vlen);
```

Reply:
I fixed this in a later commit in the series, I'll just merge the fix with the original change.

Reply:
Ah, okay, obviously I was reviewing the commits chronologically.